public/content/lessons/2016/linear-transformations/index.mdx

```diff
@@ -42,355 +36,534 @@ So why use the word “transformation” instead of “function” if they mean
 $$
 
-</FreeResponse>
+What all of this is saying is that the two-dimensional linear transformation is completely described by just four numbers: The two coordinates for where  $\hat{\imath}$ lands, and the two coordinates for where $\hat{\jmath}$ lands. Isn't that cool?
 
-If you were able to get that, then congratulations, you just reinvented matrix vector multiplication.
+<Figure
+  image="./figures/matrices/LinearTransformationFormula.svg" 
+/>
 
-You see, it’s common to package these four numbers which characterize a given transformation into a $2\times 2$ grid of numbers, called a “2-by-2 matrix”, where you can interpret the columns as the two special vectors where $\hat i$ and $\hat j$ land.
+It's common to package these four numbers into a 2x2 grid of numbers, called a “2x2 matrix”, where you can interpret the columns as the two special vectors where $\hat{\imath}$ and $\hat{\jmath}$ land.
 
-$$
-``2\times 2\text{ Matrix''} \\\\
-\begin{bmatrix}
-\color{green}1 & \color{red}3 \\
-\color{green}-2 & \color{red}0
-\end{bmatrix}
-$$
+<Figure
+  image="./figures/matrices/MatrixNotation2x2.svg" 
+/>
 
-If you’re given a 2x2 matrix describing a linear transformation, and a specific vector, and you want to know where the linear transformation takes that vector, you take the coordinates of that vector, multiply them by the corresponding column of the matrix, then add together what you get. This corresponds with the idea of adding scaled versions of our new basis vectors.
+If you're given a 2x2 matrix describing a linear transformation, and a specific vector, and you want to know where the linear transformation takes that vector, you take the coordinates of that vector, multiply them by the corresponding column of the matrix, then add together what you get. This corresponds with the idea of adding scaled versions of our new basis vectors.
 
-$$
-\begin{bmatrix}
-\color{green}3 & \color{red}2 \\
-\color{green}-2 & \color{red}1
-\end{bmatrix}
-\cdot
-\begin{bmatrix}5\\7\end{bmatrix}
-=
-5\begin{bmatrix}\color{green}3\\ \color{green}-2\end{bmatrix}
-+7\begin{bmatrix}\color{red}2\\ \color{red}1\end{bmatrix}
-=
-\begin{bmatrix}29\\-3\end{bmatrix}
-$$
-We can generalize this idea with a matrix that has variable entries:
-$$
-\begin{bmatrix}
-\color{green}a & \color{red}b \\
-\color{green}c & \color{red}d
-\end{bmatrix}
-\cdot
-\begin{bmatrix}x\\y\end{bmatrix}
-=
-x\begin{bmatrix}\color{green}a\\ \color{green}c\end{bmatrix}
-+y\begin{bmatrix}\color{red}b\\ \color{red}d\end{bmatrix}
-=
-\begin{bmatrix}
-\color{green}a\color{black}x+\color{red}b\color{black}y \\
-\color{green}c\color{black}x+\color{red}d\color{black}y
-\end{bmatrix}
-$$
-Remember that this all came from thinking about the columns as the transformed versions of your basis vectors. Then the result is the appropriate linear combination of those vectors.
+<Figure
+  image="./figures/matrices/MatrixNotation2x2InputOutput.svg" 
```

Comment:
I agree this illustration and the one below was a bit confusing. I just updated this illustration and the one below to something like this. What do you think?

<img width="625" alt="Screenshot 2024-01-30 at 12 26 36 PM" src="https://github.com/3b1b/3Blue1Brown.com/assets/12160501/2d23d284-2e45-45a6-8bd8-f4dac0ef8dcf">

Reply:
Much better, thanks!

Maybe there's not room to do it cleanly, but you could also try one where the arrow is extra long, and the matrix is small and sits on top of the arrow, as a way of communicating that it is the transform.